src/template/ADSPHelpers.cpp

```diff
@@ -441,7 +441,7 @@ float CADSPHelpers::Convert_dB_TO_Value(float dB)
 
 float CADSPHelpers::Convert_Value_TO_dB(float Scale)
 {
-  return 20.0f*log10f(abs(Scale));
+  return 20.0f*log10f(iabs(Scale));
```

Comment:
Thanks for the patch, but I guess this will break the Windows build. Because `#if !defined(TARGET_WINDOWS)`.